src/index.ts

```diff
@@ -14,6 +14,8 @@ const backendDependencies = {
 
 function setupOpenMRS() {
   const moduleName = "@openmrs/esm-form-builder";
+  const route = `form-builder`;
+  const spaBasePath = `${window.spaBase}/${route}`;
```

Comment:
If you aren't re-using the `route` variable anywhere else, just:

```js
const spaBasePath = `${window.spaBase}/form-builder`;
```

Reply:
I have used `route` in line number 29